pkg/kubelet/network/dns/dns_test.go

```diff
@@ -586,6 +586,7 @@ func TestGetPodDNSCustom(t *testing.T) {
 	testNdotsOptionValue := "3"
 	testHostNameserver := "8.8.8.8"
 	testHostDomain := "host.domain"
+	testHostOption := "ndots:1"
```

Comment:
I would suggest to use a more explicit name, since this value is not a default in all test cases.